src/wp-includes/class-wp-image-editor.php

```diff
@@ -355,32 +356,36 @@ protected function get_output_format( $filename = null, $mime_type = null ) {
 			$new_ext   = $file_ext;
 		}
 
-		/**
-		 * Filters the image editor output format mapping.
-		 *
-		 * Enables filtering the mime type used to save images. By default,
-		 * the mapping array is empty, so the mime type matches the source image.
-		 *
-		 * @see WP_Image_Editor::get_output_format()
-		 *
-		 * @since 5.8.0
-		 *
-		 * @param string[] $output_format {
-		 *     An array of mime type mappings. Maps a source mime type to a new
-		 *     destination mime type. Default empty array.
-		 *
-		 *     @type string ...$0 The new mime type.
-		 * }
-		 * @param string $filename  Path to the image.
-		 * @param string $mime_type The source image mime type.
-		 */
-		$output_format = apply_filters( 'image_editor_output_format', array(), $filename, $mime_type );
 
-		if ( isset( $output_format[ $mime_type ] )
-			&& $this->supports_mime_type( $output_format[ $mime_type ] )
-		) {
-			$mime_type = $output_format[ $mime_type ];
-			$new_ext   = $this->get_extension( $mime_type );
+		// Check if this image size doesn't support output mapping.
+		if ( empty( $_wp_additional_image_sizes[ $this->size ]['disable_output_mapping'] ) ) {
```

Comment:
this doesn't actually work because `size` is not the size name, it is the dimensions. Thinking we can add size_name to make this work.